### PR TITLE
Documentation of getting-started in Japanese

### PR DESCRIPTION
**Pull request by tac0x2a**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation

### CODE REVIEW

**Review by blinkov — APPROVED** [review]

File: docs/ja/index.md [hunk 1]
```diff
@@ -1,66 +1,62 @@
...
 
-| Row | WatchID     | JavaEnable | Title              | GoodEvent | EventTime           |
-|-----|-------------|------------|--------------------|-----------|---------------------|
-| \#0 | 89354350662 | 1          | Investor Relations | 1         | 2016-05-18 05:19:20 |
-| \#1 | 90329509958 | 0          | Contact us         | 1         | 2016-05-18 08:10:20 |
-| \#2 | 89953706054 | 1          | Mission            | 1         | 2016-05-18 07:38:00 |
-| \#N | …           | …          | …                  | …         | …                   |
+Row | WatchID | JavaEnable | Title | GoodEvent | EventTime
+--- | --- | --- | --- | --- | ---
+#0 | 89354350662 | 1 | Investor Relations | 1 | 2016-05-18 05:19:20
```
Comment (blinkov) [inline docs/ja/index.md:14]: @tac0x2a could you revert these changes to tables? It seems like they are the reason for the failed tests.

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/ja/index.md (hunk 1 above)
Comment (tac0x2a) [reply docs/ja/index.md]: @blinkov Thanks to review! I fixed it!

**Comment by blinkov** [comment]
@tac0x2a there are some broken links:
```

WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/interfaces/jdbc.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/interfaces/odbc.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/(../interfaces/index.md)' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'sql-reference/statements/create/table.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'getting-started/interfaces/index.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'getting-started/sql-reference/statements/optimize.md' which is not found in the documentation files.
```

**Comment by blinkov** [comment]
@tac0x2a there's one more in the last commit: `WARNING:mkdocs.structure.pages:Documentation file 'sql-reference/statements/optimize.md' contains a link to 'sql-reference/statements/alter/index.md' which is not found in the documentation files.`

**Comment by blinkov** [comment]
@tac0x2a there're some broken links:
```
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/interfaces/jdbc.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/interfaces/odbc.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/(../interfaces/index.md)' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'sql-reference/statements/create/table.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'getting-started/interfaces/index.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'getting-started/sql-reference/statements/optimize.md' which is not found in the documentation files.
```

**Comment by tac0x2a** [comment]
@blinkov Thank you for reporting. But, it looks same comment to [this](https://github.com/ClickHouse/ClickHouse/pull/13966#issuecomment-679861857). Is it collect ?

By the way, how can I check broken links on my PC ?

**Comment by blinkov** [comment]
@tac0x2a yes, the last commit looks green, thanks!

For testing locally see: https://github.com/ClickHouse/ClickHouse/blob/master/docs/tools/README.md
